model/ids.go

```diff
@@ -166,6 +181,15 @@ func SpanIDFromString(s string) (SpanID, error) {
 	return SpanID(id), nil
 }
 
+// SpanIDFromBytes creates a SpandID from list of bytes
+func SpanIDFromBytes(data []byte) (SpanID, error) {
```

Comment:
to follow on the TraceIDFromBytes, swap it and implement SpanID.Unmarshal in terms of SpanIDFromBytes.